### ISSUE TITLE
You should use a real licence

### ISSUE THREAD

**Issue by Secret-chest** [jeffalo/ocular#188]
CC licences (except CC0) are not for software, they only apply to art!

**Comment by RedGuy12** [comment]
literally everything on scratch is licensed under cc-by-sa 3.0 tf you mean

**Comment by PPPDUD** [comment]
Better alternative: The Unlicense.

**Comment by Secret-chest** [comment]
> Better alternative: The Unlicense.

The CC0 is better tested and works in more countries.

**Comment by PlayfulMathematician** [comment]
no cc-by-sa 3.0 is irrevocable so they can't take it off

**Comment by Secret-chest** [comment]
> no cc-by-sa 3.0 is irrevocable so they can't take it off

Yes, but you can change it to another one if it's compatible or you have permission from all the authors.

**Comment by DDCcarr1** [comment]
This isn’t an issue…next time use the Discussions tab.
